stacks/java-quarkus/devfile.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     - Quarkus
   projectType: Quarkus
   language: Java
-  version: 1.3.0
+  version: 1.3.1
```

Comment:
Hi @vinokurig,

I'd recommend also creating a separate version directory for the new version of the stack, see our `go` stack directory for how this is done: https://github.com/devfile/registry/tree/27951b4afc4feba8fab57538f0ed263d14fa5c18/stacks/go

So the outcome would be a new `stack.yaml` with both `1.3.0/devfile.yaml` and `1.3.1/devfile.yaml` versioned devfiles.

Reply:
Done